### PR TITLE
Writing mutated text as well when fuzzing in mutate fuzz-target

### PR DESCRIPTION
**Pull request by Jacarte**

### CODE REVIEW

File: fuzz/fuzz_targets/mutate.rs [hunk 1]
```diff
@@ -48,8 +49,22 @@ fuzz_target!(|inputs: (wasm_smith::Module, u64)| {
...
+    let mut validator = wasmparser::Validator::new();
+    validator.wasm_features(features);
+
+
+    let validation_result = validator.validate_all(&mutated_wasm);
     log::debug!("validation result = {:?}", validation_result);
+
+
+    let mutated_text = wasmprinter::print_bytes(mutated_wasm).unwrap();
+    std::fs::write("mutated.wat", &mutated_text).expect("should write `mutated.wat` okay");
```
Comment (fitzgen) [inline fuzz/fuzz_targets/mutate.rs:66]: Can you wrap this in a `if log::log_enabled!(log::Level::Debug) { ... }` like the other places where we write files? This should help keep fuzzer throughput pretty high when we aren't actively debugging a failing input.

File: fuzz/fuzz_targets/mutate.rs [hunk 2]
```diff
@@ -48,8 +49,22 @@ fuzz_target!(|inputs: (wasm_smith::Module, u64)| {
...
+    features.multi_value = true;
+
+    let mut validator = wasmparser::Validator::new();
+    validator.wasm_features(features);
+
+
+    let validation_result = validator.validate_all(&mutated_wasm);
     log::debug!("validation result = {:?}", validation_result);
+
+
```
Comment (fitzgen) [inline fuzz/fuzz_targets/mutate.rs:64]:
```suggestion

```

File: fuzz/fuzz_targets/mutate.rs [hunk 3]
```diff
@@ -48,8 +49,22 @@ fuzz_target!(|inputs: (wasm_smith::Module, u64)| {
...
-    let validation_result = wasmparser::validate(&mutated_wasm);
+    // Add valiation with all features enabled, like multiple value return
+    let mut features = WasmFeatures::default();
+    // Add other if needed
+    features.multi_value = true;
+
+    let mut validator = wasmparser::Validator::new();
+    validator.wasm_features(features);
+
+
```
Comment (fitzgen) [inline fuzz/fuzz_targets/mutate.rs:60]:
```suggestion

```

**Review by fitzgen — COMMENTED** [review]
LGTM with a small tweak

File: fuzz/fuzz_targets/mutate.rs (hunk 1 above)
Comment (Jacarte) [reply fuzz/fuzz_targets/mutate.rs]: Done

**Review by fitzgen — APPROVED** [review]

**Pull request merged**